driver.sh

```diff
@@ -147,6 +153,12 @@ setup_variables() {
           qemu_cmdline=( -drive "file=images/x86_64/rootfs.ext4,format=raw,if=ide"
```

Comment:
I was expecting if=virtio here. Using if=ide seems problematic. (Though it does test a rarely-used block driver!)

My VMs boot with:

        -drive file=drive.raw,if=virtio,format=raw

Probably I need to switch to the -device style (as in arm64 above). But in either way, you'd need to update the /dev/sda* to /dev/vda*

I wonder if the same failure condition is due in both cases to using a partition-less disk image?